feeds/indio/indio/files/bin/sys_led.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/sh
+
+# Maximum number of retries
+max_retries=5
+
+# Delay between retries (in seconds)
+retry_delay=1
+
+# Function to turn on the LED (green)
+turn_on_green_led() 
+{
+	echo "1" > /sys/class/leds/led_2g/brightness
+	echo "0" > /sys/class/leds/led_5g/brightness      
+	echo "0" > /sys/class/leds/led_sys/brightness
+
+}
+
+# Function to turn on the LED (red)
+turn_on_blue_led() 
+{
+	echo "1" > /sys/class/leds/led_2g/brightness
+	echo "1" > /sys/class/leds/led_5g/brightness      
+	echo "0" > /sys/class/leds/led_sys/brightness
+}
+
+turn_on_pink_led()
+{
+
+	echo "0" > /sys/class/leds/led_2g/brightness
+	echo "1" > /sys/class/leds/led_5g/brightness
+	echo "1" > /sys/class/leds/led_sys/brightness
+}
+
+# Function to check internet connectivity
+check_internet() 
+{
+	if ping -q -c 1 -W 1 8.8.8.8 > /dev/null; then
+#        	echo "Internet is working"
+        	return 0
+    	else
+#        	echo "Internet is not working"
+        	return 1
+    	fi
+}
```

Comment:
google does not allow using their infra for online detection. ucentral already has a mechanism to check for online connectivity, just set the correct led alias inside the DTS